frontend/ui/widget/numberpickerwidget.lua

```diff
@@ -14,7 +14,8 @@ local _ = require("gettext")
 local Screen = Device.screen
 
 local NumberPickerWidget = InputContainer:new{
-    spinner_face = Font:getFace("x_smalltfont",24),
+    spinner_face_font = "x_smalltfont",
```

Comment:
`smalltfont` is already 24

https://github.com/koreader/koreader/blob/47139266b2fe90e2aa6f74a9417d043e341811ec/frontend/ui/font.lua#L58

(that's actually a change I included in #3203 already so you can probably safely just leave this the way it is)